sdk/testcommon/Azure.Graph.Rbac/README.md

```diff
@@ -7,7 +7,7 @@
 * Next, add a **bulleted list** of the **most common tasks** supported by the package or library, prefaced with "Use the client library for [Product Name] to:". Then, provide code snippets for these tasks in the [Examples](#examples) section later in the document. Keep the task list short but include those tasks most developers need to perform with your package.
 * Include this single line of links targeting your product's content at the bottom of the introduction, making any adjustments as necessary (for example, NuGet instead of PyPi):
 
-  [Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/azure-batch) | [Package (PyPi)](https://pypi.org/project/azure-batch/) | [API reference documentation](https://docs.microsoft.com/python/api/overview/azure/batch?view=azure-python) | [Product documentation](https://docs.microsoft.com/azure/batch/)
+  [Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/batch/azure-batch) | [Package (PyPi)](https://pypi.org/project/azure-batch/) | [API reference documentation](https://docs.microsoft.com/python/api/overview/azure/batch?view=azure-python) | [Product documentation](https://docs.microsoft.com/azure/batch/)
```

Comment:
If possible, can we point these at Azure.Graph.Rbac instead of batch?  It's fine to merge in like this if we can't find that location quickly.

Reply:
Ditto for the rest of these README templates that all point to batch.

Reply:
For management plane README we have the issue tracking here - https://github.com/Azure/azure-sdk-for-net/issues/13931

For rest of the data-plane libraries, these libraries are still new and not published yet. So they should be replaced with the actual information related to the library. In the template README, it is more like an example